### PR TITLE
Displaying all event data

### PR DESCRIPTION
**Pull request by madelinemacd**
I added a few extra event details and what columns they map to for display purposes, and removed initializing each event with empty values to clean up the details pane.

### CODE REVIEW

**Comment by madelinemacd** [comment]
I added a little bit of code to account for errors when clicking on events with empty text fields. This also solves the problem where the text tab was highlighted after clicking over an event with an empty text field. This isn't a permanent solution, I'll make more substantial changes when session templates get implemented for the June release.

File: src/sql/parts/insights/browser/insightsDialogView.ts [hunk 1]
```diff
@@ -168,13 +168,16 @@ export class InsightsDialogView extends Modal {
...
+
 		this._topTableData = new TableDataView();
 		this._bottomTableData = new TableDataView();
-		let topTableView = new TableCollapsibleView(nls.localize("insights.dialog.items", "Items"), { sizing: ViewSizing.Flexible, ariaHeaderLabel: 'title' }, this._topTableData, this._topColumns, { forceFitColumns: true });
+		let topTableView = new TableCollapsibleView(itemsHeaderTitle, { sizing: ViewSizing.Flexible, ariaHeaderLabel: itemsHeaderTitle }, this._topTableData, this._topColumns, { forceFitColumns: true });
 		this._topTable = topTableView.table;
 		topTableView.addContainerClass('insights');
 		this._topTable.setSelectionModel(new RowSelectionModel<ListResource>());
-		let bottomTableView = new TableCollapsibleView(nls.localize("insights.dialog.itemDetails", "Item Details"), { sizing: ViewSizing.Flexible, ariaHeaderLabel: 'title' }, this._bottomTableData, this._bottomColumns, { forceFitColumns: true });
+		let bottomTableView = new TableCollapsibleView(itemsDetailHeaderTitle, { sizing: ViewSizing.Flexible, ariaHeaderLabel: itemsDetailHeaderTitle }, this._bottomTableData, this._bottomColumns, { forceFitColumns: true });
```
Comment (MattIrv) [inline src/sql/parts/insights/browser/insightsDialogView.ts]: Thanks for fixing these

File: src/sql/parts/profiler/editor/profilerInput.ts [hunk 2]
```diff
@@ -156,12 +155,12 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
...
+			columnNameMap['client_pid'] = 'ClientProcessID';
+			columnNameMap['writes'] = 'Writes';
+			columnNameMap['duration'] = 'Duration';
 
-			for (let idx = 0; idx < columns.length; ++idx) {
-				let columnName = columns[idx];
-				data[columnName] = '';
-			}
-
+			data['TextData'] = ' ';
```
Comment (MattIrv) [inline src/sql/parts/profiler/editor/profilerInput.ts]: Why `' '`? Would `''` (no space) or `undefined` be better?

From the code alone it's not really clear why we need this so I'd suggest adding a comment in any case

Comment (madelinemacd) [reply src/sql/parts/profiler/editor/profilerInput.ts]: So it turns out that having undefined crashes the whole thing, and adding a space solves the issue with highlighting text as you scroll through. It looks like the issue was purely when the text field is completely empty it highlights the next displayed text, but a single space fixes it. It's definitely a hacky work around, but I figured it improves the profiler experience and is probably good enough for the June release. I'll be changing a lot of display logic for the July release, so I'm planning to tackle this more permanently then.

Comment (MattIrv) [reply src/sql/parts/profiler/editor/profilerInput.ts]: Sounds good

**Pull request merged**